tg_bot/modules/users.py

```diff
@@ -65,6 +68,136 @@ def broadcast(bot: Bot, update: Update):
+    new_members = update.effective_message.new_chat_members
+    user = update.effective_user
+
+    if sql.get_restriction(chat.id):
+        if not user.id in SUDO_USERS:
+            if any(new_mem.id == bot.id for new_mem in new_members):
+                update.effective_message.reply_text("I have been restricted by my admins from this chat! "
+                                                    "Request any of my admins to add me to this chat.")
+                bot.leave_chat(chat.id)
+
+        # Unrestrict group if a sudo user adds bot
+        else:
+            if any(new_mem.id == bot.id for new_mem in new_members):
+                sql.set_restriction(chat.id, restricted = False)
+
+                # Report to sudo users
+                send_to_list(bot, SUDO_USERS,
+                             "{} has added me to the chat <b>{}</b> and removed my restrictions."
+                             .format(mention_html(user.id, user.first_name), chat.title),
+                             html=True)
+
+@run_async
+def unrestrict_group(bot: Bot, update: Update, args: List[str]) -> str:
+    message = update.effective_message
+    
+    # Check if there is only one argument
+    if len(args) == 1:
+        chat_id = args[0]
+
+        # Check if chat_id is valid
+        if chat_id.startswith('-'):
+            # Check if chat_id is in bot database
+            all_chats = sql.get_all_chats() or []
+            if any(chat.chat_id == chat_id for chat in all_chats):
+                chat_restricted = sql.get_restriction(chat_id)
+                if chat_restricted:
+                    sql.set_restriction(chat_id, restricted = False)
+            
+                    message.reply_text("Successfully removed all restrictions on the chat with id `{}`"
+                                       .format(chat_id), parse_mode = ParseMode.MARKDOWN)
+                
+                    # Report to sudo users
+                    unrestrictor = update.effective_user  # type: Optional[User]
+                    send_to_list(bot, SUDO_USERS,
+                                 "{} has removed my restrictions on the chat with id <code>{}</code>."
+                                 .format(mention_html(unrestrictor.id, unrestrictor.first_name), chat_id),
+                                 html=True)
+            
+                else:
+                    message.reply_text("I'm not restricted from that chat!")
+        
+            else:
+                message.reply_text("I can't seem to find the chat in my database. "
+                                   "Use /chatlist to obtain a list of chats in my database.")
+        
+    
+        else:
+            message.reply_text("Invalid chat id! Make sure you include the '-' sign in the chat id.")
+
+    else:
+        message.reply_text("Incorrect number of arguments. Please use `/unrestrict chat_id`.",
+                           parse_mode = ParseMode.MARKDOWN)
```

Comment:
PEP8 - going to mention it again, `parse_mode=ParseMode.MARKDOWN`, no spaces around the = when using named arguments.